provider/win_dump_utils.py

```diff
@@ -23,7 +23,8 @@ def set_vm_for_dump(test, params):
     sys_image_size = int(float(
             utils_misc.normalize_data_size(params["image_size"], "G")))
     if host_avail_disk < (host_free_mem // 1024**2) * 1.2 + sys_image_size:
-        params["mem"] = (host_avail_disk - sys_image_size) * 0.8 // 2.4 * 1024
+        avail_dump_size = host_avail_disk - sys_image_size
+        params["mem"] = str(int((avail_dump_size) * 0.8 // 2.4) * 1024)
```

Comment:
What do you think about `utils_misc.normalize_data_size("%sM" % mem_params["mem"], "G")) * 1.4)`?

Reply:
@vivianQizhu Updated, thanks~